scripts/jenkins/Makefile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Makefile for jenkins-tests intel-device-plugins.
+
+tests = $(shell ls -d */ | cut -f1 -d'/')
+
+REG?=intel/
+ORG?=$(REG:/=)
+TAG?=devel
+
+set-version:
+ifneq ($(ORG)$(TAG),inteldevel)
+	@bash ../set-version.sh --tag $(TAG) --org $(ORG)
```

Comment:
would it be better in `Jenkinsfile`?

Reply:
I'm keen on letting single line make commands how is trying to be. IMHO Jenkinsfile seems cleaner.

Reply:
there's actually (again) a better way to do this which just occured to me. we can configure the cluster nodes to be aware of the registry URL we are using (e.g., with cri-o it's registry.search in registry.conf).

I believe with that the original `set-version.sh` would work and we only have to change the TAG.

Reply:
This need some rethinking and discussions, I'm sending you an email with the findings using registry.search.